extensions/bitwarden/src/components.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  showToast,
+  ToastStyle,
+  Form,
+  ActionPanel,
+  SubmitFormAction,
+  Detail,
+  CopyToClipboardAction,
+  getPreferenceValues,
+} from "@raycast/api";
+import { BitwardenApi } from "./api";
+
+export function TroubleshootingGuide(): JSX.Element {
+  showToast(ToastStyle.Failure, "Bitwarden CLI not found");
+  const { path } = getPreferenceValues();
+  const markdown = `# The Bitwarden CLI was not found
+## Please check that:
+
+1. The Bitwarden CLI is [correctly installed](https://bitwarden.com/help/article/cli/#download-and-install)
+1. The path of the installation matches the Bitwarden CLI Installation Path extension setting
+> Currently set to: \`${path}\`
```

Comment:
```suggestion
  > Currently set to: \`${path}\`
```
We should indent this to confirm that it's a note related to the second list item. I _think_ 1 indentation is enough, but it might require 2 indentations to render as expected.